src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionAzureKeyVaultProvider.java

```diff
@@ -124,56 +198,43 @@ public SQLServerColumnEncryptionAzureKeyVaultProvider(
     /**
      * Constructs a SQLServerColumnEncryptionAzureKeyVaultProvider with a callback function to authenticate to AAD. This
      * is used by KeyVaultClient at runtime to authenticate to Azure Key Vault.
-     * 
+     *
      * @param authenticationCallback
      *        - Callback function used for authenticating to AAD.
      * @throws SQLServerException
      *         when an error occurs
      */
+    @Deprecated
```

Comment:
No need to deprecate this constructor anymore, because authenticationCallback works with MSAL now.